Feature Request: Add terraform export to end of Self Service Step 3

jaydhulia opened the pull request:
This PR addresses #204:

- [X] Added new pane `Terraform Exporter` to Self Service Step 3
- [X] The pane `Terraform Exporter` displays the Terraform version of the IAM policy in a read-only editor
- [X] Added configuration variable on the backend for enabling or disabling terraform exporter `export_to_terraform_enabled`
- [X] The pane `Terraform Exporter` is only visible when export_to_terraform_enabled = True, by default no changes will be visible

Sample of what the pane looks like:
![image](https://user-images.githubusercontent.com/46459060/107795381-d1691e00-6d15-11eb-8422-176adc89bf88.png)

jaydhulia committed:
add terraform export - initial commit

jaydhulia committed:
Delete .eslintcache

castrapel committed:
Merge branch 'master' into jdhulia-issue-204

castrapel commented:
This is rad, thanks Jay! 

We're actually looking to migrate away from ACE and use Monaco instead. Would you be able to convert the code editor in Step 3 to Monaco and implement this also in Monaco?

jaydhulia committed:
Move to Monaco instead of Ace

jaydhulia committed:
Merge branch 'jdhulia-issue-204' of github.com:jaydhulia/consoleme into jdhulia-issue-204

jaydhulia committed:
Merge branch 'master' into jdhulia-issue-204

jaydhulia commented:
- [X] Updated Step 3 editor to use Monaco, replacing Ace - Addresses issue #8934 
- [X] Updated Terraform exporter to use Monaco, instead of Ace
- [X] Fixed a bug that was causing duplicate results and requests to backend on autocomplete

jaydhulia committed:
added copy button for terraform

jaydhulia commented:
Added Copy button for copying the Terraform policy
![image](https://user-images.githubusercontent.com/46459060/107863123-ce9e2400-6e0e-11eb-9ffe-d928a8edbd98.png)

castrapel committed:
Merge branch 'master' into jdhulia-issue-204

castrapel reviewed (approved):
Looks perfect, type-aheads work as expected. Thanks Jay!
